variants/NUCLEO_L053R8/PeripheralPins.c

```diff
@@ -188,3 +188,11 @@ const PinMap PinMap_SPI_SSEL[] = {
 
 //*** CAN ***
 // NO CAN
+
```

Comment:
PeripheralPins should only contains array For Pin <--> Peripheral.
How this array is defined? Is it linked to boards? MCU ? Is it exhaustive?